config/secrets.yml

```diff
@@ -19,7 +19,7 @@ test:
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production: &production
-  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+  secret_key_base: <%= ENV.fetch('SECRET_KEY_BASE', '6d5987d15ed5f173ea13ee0a26ba1cb1717a86783177168ec42cebbbc563e0c3470be82eba91700659589a76ad83b2e3af5ea47f083290d563c2b321956ae964') %>
```

Comment:
This seems pretty scary since it could mistakenly lead to using this key in production (although I don't think we're actually using this for anything in this app). How bad is it to specify `SECRET_KEY_BASE=secret` when running locally?